tests/ui/inline-const/expr-in-item-context.stderr

```diff
@@ -0,0 +1,8 @@
+error: expected item, found keyword `const`
+  --> $DIR/expr-in-item-context.rs:9:1
+   |
+LL | const { assert!(size_of::<u32>() <= size_of::<usize>()) };
+   | ^^^^^ if you meant to create an anonymous const, use `const (): _ = {};` instead
```

Comment:
```suggestion
   | ^^^^^ if you meant to create an anonymous const, use `const _: () = {};` instead
```

Reply:
...I have no idea how I got that backwards, thank you.

Reply:
Maybe we do need `const { }` items after all :D